lib/party_foul.rb

```diff
@@ -78,9 +78,9 @@ def self.repo_url
   # Will also setup for GitHub api connections
   #
   # @param [Block]
-  def self.configure(&block)
+  def self.configure
     yield self
-    self.github ||= Octokit::Client.new access_token: oauth_token, api_endpoint: api_endpoint
```

Comment:
The reason for the commit was to get the tests to pass. One test does modify the `oauth_token` at runtime, since with random test order, it could be set to something else.

But yes, if someone had multiple `oauth_tokens` set up, this change would allow that to work.

I doubt anyone does that. This change would not hurt performance if they set it up as recommended in the readme.